### PR TITLE
Bump prettier

### PR DESCRIPTION
**Pull request by azz**
Includes https://github.com/prettier/prettier/pull/3575
Unblocks #9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "node": ">=6"
   },
   "dependencies": {
-    "prettier": "azz/prettier#f6d0b5fe"
+    "prettier": "prettier/prettier#ee148bfdeda373445ac9b67338cbae3df10a6576"
```
Comment (lipis) [inline package.json]: maybe `prettier/prettier#master` since this is under dev anyway?

Comment (azz) [reply package.json]: Better to be explicit in package.json since it's locked in `yarn.lock` anyway.

**Comment by vjeux** [comment]
I had to reset the travis cache as it was using a different version for some reason. By the way, why does travis run the tests twice, once for "pr" and once for "push"?

**Comment by j-f1** [comment]
> By the way, why does travis run the tests twice, once for "pr" and once for "push"?

PR builds are run on [a merge commit](https://github.com/prettier/prettier-python/commit/46f4bf0e69086ead5c696b0e4e48e744d4fe123d) automatically created on each update of the PR, merging it into the target branch.

Push builds are run on any commit pushed to the repository.

**Comment by taion** [comment]
It's because you're submitting PRs from the main repo.

You need to set `branches` in `.travis.yml`: https://github.com/react-bootstrap/react-bootstrap/blob/b6fb8d6e7f34d6849de19211537912c188a604ff/.travis.yml#L21-L24

This will disable "push" builds on branches other than the ones listed above.

**Comment by azz** [comment]
Yeah I pushed the branch to the wrong remote (forgot the `-u`)

**Comment by taion** [comment]
In general it's not a bad idea to just whitelist master/next though

**Pull request merged**